### ISSUE TITLE
Some fields not announced in NVDA

### ISSUE THREAD

**Issue by hidde** [w3c/wai-atag-report-tool#47]
(bug reported via email by Sylvie, to be investigated)

**Comment by hidde** [comment]
This was due to fields sharing the same IDs, fixed in https://github.com/w3c/wai-atag-report-tool/commit/5ba63e87e84920cc8b167f4cf03c4228e1213287.
